### PR TITLE
FTC Style selection

### PR DESCRIPTION
**Pull request by lukefrogger**
## What's here?
- Updates to handle more complex styles for use in the Kadence template selection.
- A skeleton loading state

### Screenshots
Screenshot 2023-05-17 at 1 12 19 PM

Screenshot 2023-05-17 at 1 12 25 PM

### CODE REVIEW

File: packages/sitebuilder/src/wizards/first-time-configuration/components/KadenceTemplateItem.tsx [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
 const embedWidth = 1280;
 const shadowRootStyles = {
-	position: 'absolute',
 	width: embedWidth,
 	transform: 'scale(0.2)',
 	transformOrigin: '0 0',
 	border: 0,
 	overflow: 'hidden'
 };
+// type OneOnly<Obj, Key extends keyof Obj> = { [key in Exclude<keyof Obj, Key>]: null } & Pick<Obj, Key>;
```
Comment (chumpgrub) [inline packages/sitebuilder/src/wizards/first-time-configuration/components/KadenceTemplateItem.tsx:19]: Can this be removed?

File: packages/sitebuilder/src/wizards/first-time-configuration/screens/StyleSelection.tsx
Comment (chumpgrub) [inline packages/sitebuilder/src/wizards/first-time-configuration/screens/StyleSelection.tsx:1]: 😍

**Review by chumpgrub — APPROVED** [review]
This looks real nice! I just had one question about a commented-out type definition.

**Review by chumpgrub — APPROVED** [review]
This looks real nice! I just had one question about a commented-out type definition.

File: packages/sitebuilder/src/wizards/first-time-configuration/components/KadenceTemplateItem.tsx (hunk 1 above)
Comment (lukefrogger) [reply packages/sitebuilder/src/wizards/first-time-configuration/components/KadenceTemplateItem.tsx:19]: Good catch! Will remove 👍

**Pull request merged**